### DISCUSSION TITLE
Exposing countries and states in the API

### DISCUSSION THREAD

**Discussion by vimtor** [swellstores/community#159] [Help] [answered]
I am building a custom storefront and when implementing the register form I needed to get the customer country and state information.
However I could not find the API endpoint for this so I had to reverse-engineer the code to get the huge JSON of countries and states with their respective values.

I guess this could be exposed in the API similar to currencies or other metadata values :)

**Comment by ericingram** [comment, marked as answer]
This data is available from the backend API, for example:

```
const { countries, states, timezones, locales, currencies } = await swell.get('/settings/geo')
```

Good idea to make this available from the frontend SDK too!

**Reply by vimtor** [reply]
Thank you!
